ldms/man/ldmsd_sampler_advertisement.man

```diff
@@ -47,7 +47,7 @@ ldmsd_sampler_advertisement - Manual for LDMSD Sampler Advertisement
 
 .SH DESCRIPTION
 
-LDMSD Sampler Discovery is a capability that enables LDMSD automatically add
+LDMSD Sampler Advertisement is a capability that enables LDMSD automatically add
```

Comment:
LDMSD Peer Daemon advertisement is a capability that enables LDMSD to automatically add